helm/manager/values.yaml

```diff
@@ -47,11 +47,11 @@ ingress:
 
 resources:
   limits:
-   cpu: 1000m
+   cpu: 2000m
    memory: 2Gi
   requests:
-   cpu: 1000m
-   memory: 1Gi
+   cpu: 2000m
```

Comment:
Why we need so much memory and cpu in requests? imho 1 cpu and 1Gi would be enough

Reply:
Because it's a configuration that worked on my PC.
I'm not sure why previous requests wasn't enough :man_shrugging: